loops_lab.js

```diff
@@ -0,0 +1,154 @@
+//Loops lab Assignment
+
+//1.
+// let nums = 30
+// while (nums >= 0) {
+//   console.log(nums)
+//   nums -= 1
+// }
+
+// for (let nums = 30; nums >= 0; nums -= 1) {
+//   console.log(nums)
+// }
+
+
+// //2.
+// let max = 0;
+// while (max <= 30) {
+//   max % 2 !== 0
+//   ? console.log(max + ' is odd')
+//   : console.log(max + ' is even');
+//   max += 1;
+// }
+
+
+// for (let pNum = 0; pNum <= 30; pNum++) {
+//   if (pNum % 2 !== 0) {
+//     console.log(pNum + ' is odd')
+//     } else {console.log(pNum + ' is even')
+//   }
+// }
+
+
+//3.
+// let thisNum = 1
+// let multNum = 9
+//
+// while(thisNum <= 9) {
+//     let resNum = thisNum * multNum;
+//     console.log(thisNum + ' * 9 = ' + resNum);
+//     thisNum++;
+// }
+
+// for (let thisNum = 1; thisNum <= 9; thisNum++) {
+//   for (let multNum = 9; multNum <= 9; multNum++) {
+//     let resNum = thisNum * multNum;
+//     console.log(thisNum + ' * 9 = ' + resNum);
+//   }
+// }
+//DIFFERENT WAYS TO SOLVE QUESTION
```

Comment:
Good use of modulo to solve the problem differently. It is always good idea to think about different ways to solve the same problem.